config.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  log-level:
+    description: |
+      Configures the log level of synapse.
+
+      Acceptable values are: "info", "debug", "warning", "error" and "critical"
+    default: "info"
+    type: string
+  server_name:
+    description: |
+      Synapse server name.
```

Comment:
Similar comment as earlier, should the descriptions in this file also be updated to be a bit more informative? Is this supposed to be the name of the DNS record? Or something else?

Reply:
This description could link to https://matrix-org.github.io/synapse/latest/setup/installation.html#choosing-your-server-name

Reply:
Makes sense, I update the description and added an example of a valid server name.

Reply:
I feel like this config option does not make justice of the fact that the server name cannot be changed later. It should probably be retrieved from a configuration file instead.

Reply:
the charm uses this as a last resort, SYNAPSE_SERVER_NAME env takes precedence